yarn-project/aztec/package.json

```diff
@@ -53,6 +53,7 @@
     "@aztec/pxe": "workspace:^",
     "@aztec/telemetry-client": "workspace:^",
     "@aztec/txe": "workspace:^",
+    "@aztec/types": "workspace:^",
```

Comment:
I got bit by it yesterday, this needs a `yarn prepare` at the root to update `aztec/tsconfig.json` :face_holding_back_tears:

Reply:
I did run it 🤔

Reply:
yep it's there 👍